build/ci/config_files/prometheus.yaml

```diff
@@ -0,0 +1,20 @@
+#  Copyright (C) 2025 B3Partners B.V.
+#  SPDX-License-Identifier: MIT
+global:
+  # very short interval for development purposes
+  scrape_interval: '5s'
+
+scrape_configs:
+  - job_name: 'tailormap-api-snapshot'
+    scheme: 'http'
+    basic_auth:
+      password: 'tm-admin'
+      username: 'tm-admin'
+    static_configs:
+      - targets:
+        - 'host.docker.internal:8080'
+    metrics_path: '/api/actuator/prometheus'
+    params:
```

Comment:
Maybe we can use use relabel_config with a regexp to filter by prefix instead of having to name all metrics,

https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
```
    metric_relabel_configs:
    - source_labels: [__name__]
      action: keep
      regex: '^tailormap.*'
```
(untested)